doc/connectivity/bluetooth/api/index.rst

```diff
@@ -40,3 +37,37 @@ Bluetooth APIs
    shell/mcp.rst
    shell/tmap.rst
    shell/pbp.rst
+
+Bluetooth LE Host
+=================
+
+.. toctree::
+   :maxdepth: 1
+
+   services.rst
+   gap.rst
+   gatt.rst
+   att.rst
+
+Core Host and drivers
+=====================
+
+.. toctree::
+   :maxdepth: 1
+
+   l2cap.rst
+   connection_mgmt.rst
+   data_buffer.rst
+   hci_drivers.rst
+   hci_raw.rst
+   crypto.rst
+
+Other
+=====
+
+.. toctree::
+   :maxdepth: 1
+
+   mesh.rst
```

Comment:
Hah, poor mesh, being reduce to a single point in "Other" :)

Reply:
I really didn't know where to put it :/

Reply:
I get that. I added the maintainer of the Mesh (@PavelVPV) as a reviewer as well. I think they should have a say in how Mesh should be handled in the documentation as well.

Reply:
yepp. keep im mind  @PavelVPV that I will have other doc PRs throughout this and next weeks. So the discussion on how to best handle and showcase mesh could go in a mesh-specific PR later :)

Reply:
Why not put mesh under "Bluetooth Mesh" section?

Reply:
Then I need to make a controller section also with only one link?

Reply:
Thanks! Maybe it is the time for mesh team to revise the mesh upstream documentation.